### ISSUE TITLE
[Feature Request] ### The Big, Please Add My Provider/Client Thread ### 

### ISSUE THREAD

**Issue by neoatomic** [SiCKRAGETV/sickrage-issues#1741]
As the Issue tracker is rapidly getting filled its getting harder and harder to have a clear overview. 
To counter this we are going to collect all the provider and client requests in this thread.

To request a provider please post the following. :

Name :
URL : 
What kind of provider.? : (Torrent, NZB, Subtitle, Indexer, download client.)
Open or Invite only? : 
Does the website have a API available.? :  
Is it language specific.? : (focused on a single language or country.)
Motivate why this provider needs to be added.? : 

Note: If you request a  "invite only"  provider than we need an invite.! Those can be sent to sickrage.tv@gmail.com 

Open Torrent providers. :
Add getstrike.net https://github.com/SiCKRAGETV/sickrage-issues/issues/1000 (API: https://getstrike.net/api/)
Add bitsnoop https://github.com/SiCKRAGETV/sickrage-issues/issues/81 (API: http://bitsnoop.com/info/api.html )
Add torrentz.eu https://github.com/SiCKRAGETV/sickrage-issues/issues/874
Add cpasbien.pw https://github.com/SiCKRAGETV/sickrage-issues/issues/1534 (French)
Add ilcorsaronero.info https://github.com/SiCKRAGETV/sickrage-issues/issues/1688 (Italian)

Private Torrent providers. :
Add demonoid.pw https://github.com/SiCKRAGETV/sickrage-issues/issues/81 (invite only)
Add btnext.com https://github.com/SiCKRAGETV/sickrage-issues/issues/223 (invite only, Portuguese)
Add greekleech.info https://github.com/SiCKRAGETV/sickrage-issues/issues/223 (invite only, Greek)
Add trellas.eu https://github.com/SiCKRAGETV/sickrage-issues/issues/223 (Not working?)
Add baconbits.org https://github.com/SiCKRAGETV/sickrage-issues/issues/506 (invite only)
Add bitmetv.org https://github.com/SiCKRAGETV/sickrage-issues/issues/533 (invite only)
Add tTN (tntracker.org?) https://github.com/SiCKRAGETV/sickrage-issues/issues/533 (invite only, German)
Add ezrss.ch https://github.com/SiCKRAGETV/sickrage-issues/issues/1121 (Down.?)
Add torrentshack.me https://github.com/SiCKRAGETV/sickrage-issues/issues/1447 (invite only)
Add libertalia.me https://github.com/SiCKRAGETV/sickrage-issues/issues/1534 (invite only, French)
Add xthor.bz https://github.com/SiCKRAGETV/sickrage-issues/issues/1534 (invite only, French)
Add so-torrent.com https://github.com/SiCKRAGETV/sickrage-issues/issues/1534 (invite only, French)
Add norbits.net https://github.com/SiCKRAGETV/sickrage-issues/issues/1656 (invite only, Norway)
Add TitansOf.TV https://github.com/SiCKRAGETV/sickrage-issues/issues/1674 (invite only)

Current Torrent provider extensions :
Login ability for kickass.to https://github.com/SiCKRAGETV/sickrage-issues/issues/1200
Login ability for fnt.nu https://github.com/SiCKRAGETV/sickrage-issues/issues/1346
Login ability for lbertalia.me https://github.com/SiCKRAGETV/sickrage-issues/issues/1347

NZB Providers.(Open) :

nzbclub https://github.com/SiCKRAGETV/sickrage-issues/issues/270 
nzbindex https://github.com/SiCKRAGETV/sickrage-issues/issues/270 (RSS feeds are unstable and often down.)

NZB Providers.(Private) :
usenetserver.com https://github.com/SiCKRAGETV/sickrage-issues/issues/398 (paid subscription only)
nzb.to https://github.com/SiCKRAGETV/sickrage-issues/issues/270 (German and Closed.)

Subtitle Providers :
Subscene.com https://github.com/SiCKRAGETV/sickrage-issues/issues/339 (API: unknown)
italiansubs.net https://github.com/SiCKRAGETV/sickrage-issues/issues/342 (Italian subs only, API: https://api.italiansubs.net/)
podnapisi https://github.com/SiCKRAGETV/sickrage-issues/issues/459 ( API: new api not available yet)
sous-titres.eu https://github.com/SiCKRAGETV/sickrage-issues/issues/819 (French susb only API: Unknown)
Subtitulos.es https://github.com/SiCKRAGETV/sickrage-issues/issues/1002 (Spanish API: unknown)
thesubdb.com  free, subtitle database intended for opensource softwares  (API: http://thesubdb.com/api/ )

Current Subtitle Providers extentions:
Opensubtitles Login https://github.com/SiCKRAGETV/sickrage-issues/issues/1443

Index Providers : 
Add AniDB support https://github.com/SiCKRAGETV/sickrage-issues/issues/1 (API: http://wiki.anidb.net/w/API)
Add IMDB watchlist https://github.com/SiCKRAGETV/sickrage-issues/issues/309
Add thesportsdb.com https://github.com/SiCKRAGETV/sickrage-issues/issues/872 (API: thesportsdb.com/api/)

Notification Providers :
MediaBrowser notification support https://github.com/SiCKRAGETV/sickrage-issues/issues/8
Twitter notification support https://github.com/SiCKRAGETV/sickrage-issues/issues/542 (API: https://dev.twitter.com/overview/api)

Download Clients : 
put.io https://github.com/SiCKRAGETV/sickrage-issues/issues/337 (paid subscription only)
justseed.it https://github.com/SiCKRAGETV/sickrage-issues/issues/745 (paid subscription only)
Deluge addon LabelPlus https://github.com/SiCKRAGETV/sickrage-issues/issues/987 (http://forum.deluge-torrent.org/viewtopic.php?f=9&t=42629)

Up-todate 26-05-2015

**Comment by Trapizomba** [comment]
Please, add http://www.subtitles4free.net/ as a Subtitle Provider. Thank you!

**Comment by boyahr** [comment]
Name : BtDigg
URL : https://btdigg.org/
What kind of provider.? : Torrent
Open or Invite only? : Open
Does the website have a API available.? Yes 
Is it language specific.? : No
Motivate why this provider needs to be added.? : Quality torrents with seeders and complete shows/seasons

**Comment by papampi** [comment]
Name : ettv
URL : http://extratorrent.cc/
What kind of provider.? : Torrent
Open or Invite only? : open
Does the website have a API available.? : dont know
Is it language specific.? : no
Motivate why this provider needs to be added.? : ettv is the biggest tv provider since eztv hacked.

**Comment by Hellowlol** [comment]
@neoatomic Have you got a invite to norbits?

**Comment by neoatomic** [comment]
@Hellowlol No, i haven't. In case you have a spare one just sent it to the email address in the first post. 
Once there is enough time to get providers added, the invite will be at hand.

**Comment by Hellowlol** [comment]
Ill see if i have one. Ill ping you when/if as the invites have a time limit.

**Comment by Hellowlol** [comment]
I might even do that provider if i get the time. What provider is the best to use as a template?

**Comment by neoatomic** [comment]
@Hellowlol That would be more than welcome if you are able.
Regarding the template i cant tell. But If norbit uses an API than there is a good chance you can reuse/modify one of the exciting torrent providers templates. 

Personally i'm of the opinion that the Dev's should not put time in closed providers and only focus on open providers. I find adding closed providers more a responsibility of the community. 
Those providers are for a small group of users, and even smaller if they are language specific. 
Not even mentioning that you need invites etc. to be even able to implement/troubleshoot.

**Comment by Hellowlol** [comment]
Np, ill do it myself then. They have a decent api, i just have to find to figure out what categories they used as there are no docs :P. They use the same categories as the do on the site so it shouldnt be hard.

**Comment by noodles101** [comment]
Please add:

http://www.tv4user.de/
http://www.subcentral.de/

as German subtitles providers

**Comment by neoatomic** [comment]
@noodles101 We are not adding subtitle providers our-selfs. You have to request those directly at subliminal. : https://github.com/Diaoul/subliminal

**Comment by noodles101** [comment]
@neoatomic ok done. :smile:

**Comment by daveyboy37** [comment]
Name    TVChaosuk
URL       https://tvchaosuk.com
What kind of provider.? : Torrent
Open or Invite only? : Invite
Does the website have a API available.? No 
Is it language specific.? : No
Motivate why this provider needs to be added.? : A huge back catalogue of full season packs of UK shows of all genres, from the 60's to present day.

**Comment by Anytia** [comment]
Hi, is it possible to add https://eztv.ag as provider? It's working fine for me as torrent provider.

Thanks!

**Comment by OmgImAlexis** [comment]
@Anytia that one would have to be confirmed before anyone's going to touch it. If it's still the infected site NOPE! but if it's back again I'm almost certain we can just reuse the old code we had before it was taken over.

**Comment by duramato** [comment]
@neoatomic btdigg.org ,getstrike.net , tvchaosuk.com , tTN ,xthor.bz and TitansOf.TV are now in sickrage they can be removed of this thread or maybe add like a checkbox

**Comment by starkillerOG** [comment]
Is https://torrentz.eu/ still going to be added as a free torrent provider?
There were some troubles with that provider (blocked), but as far as i see they have all been resolved by now.

**Comment by miigotu** [comment]
Soon @starkillerOG. The code is almost done anyways.

**Comment by starkillerOG** [comment]
@miigotu Thanks for the responce and hard work you guys put into this awsome program!

**Comment by austinholbrook** [comment]
As far as I can tell, HD-Bits.com isn't available as a provider in Sickrage, but it's checked off in this list. HD-Bits.com is different than the HDBits that's already included.
